documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+
+.{ocp-name} Console native SDK fetch, post, and delete calls
+
+The `fetch`, `post`, and `delete` calls in the Forklift UI are now native to the {ocp-name} API. link:https://issues.redhat.com/browse/MTV-365[(MTV-365)]
+
+.{project-short} UI plugin uses only a dynamic plugin for the SDK REST API `fetch` methods
+
+The UI plugin uses a dynamic plugin for the SDK REST API when using the `fetch` methods. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+.Migration success status bar chart 
+
+A bar chart shows the number of migrations and their status - `starts`, `finished`, and `failed`. link:https://issues.redhat.com/browse/MTV-770[(MTV-770)]
+
+.Improved list of VMs in the detailed view of each provider
+
+The detailed view of each provider includes specific information relevant to the VMs of that provider. link:https://issues.redhat.com/browse/MTV-772[(MTV-772)]
+
+.Edit plans that failed to migrate VMs
+
+Plans that have failed to migrate any VMs can be edited. Some plans fail or are canceled because the network and storage mappings are wrong. These plans can be edited until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
+
+.ESXi provider host configuration
+
+ESXi, as a vSphere provider, does not need credentials in its setup. The credentials requirements field is removed from ESX settings. link:https://issues.redhat.com/browse/MTV-793[(MTV-793)]
+
+[id="ova-provider-26_{context}"]
+=== OVA Provider
+
+This is a collection of new features for the {project-short} OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+.Get firmware detection by virt-v2v
+
+A `virt-v2v` that converts a guest also produces an OFV with configuration based on the virtual disk, such as driver or firmware. `forklift-controller` processes the produced OFV's status of the conversion pod. link:https://issues.redhat.com/browse/MTV-759[(MTV-759)]
+
+.Delete PV when the provider is deleted
+
+Each OVA provider has an attached PV and PVC. The PVC deletes automatically when the provider is deleted. {project-short} now also deletes the PV when the provider is deleted. link:https://issues.redhat.com/browse/MTV-848[(MTV-848)]
+
+.End-to-end test for import OVA
+
+The {project-short} process performs an end to end test of the imported OVA. link:https://issues.redhat.com/browse/MTV-624[(MTV-624)]
+
+.Create OVA provider under all projects
+
+The OVA provider can be created under all projects, not only under the forklift or MTV project. link:https://issues.redhat.com/browse/MTV-681[(MTV-681,] link:https://issues.redhat.com/browse/MTV-715[MTV-715)]
+
+.Migrate from an OVA provider to a restricted project
+
+Migration from OVA provider to a restricted project succeeds past the `kubevirt` step. link:https://issues.redhat.com/browse/MTV-689[(MTV-689)]
```

Comment:
Generally, it feels like this release note misses some details..Is it possible to make the title more new-feature-like and also add some details to the description? 

```suggestion
.Migrate from an OVA provider to a restricted project

With this update, migration from the Open Virtual Appliance (OVA) provider to a restricted project now succeeds past the `kubevirt` step. link:https://issues.redhat.com/browse/MTV-689[(MTV-689)]
```

Reply:
Added 'now'